src/tests/integration-test

```diff
@@ -997,34 +1041,40 @@ class FS(UDisksTestCase):
         # https://github.com/karelzak/util-linux/issues/17
         # https://github.com/karelzak/util-linux/issues/18
         if type not in ['reiserfs', 'xfs']:
-            # the scsi_debug CD drive content is the same as for the HD drive, but
-            # udev does not know about this; so give it a nudge to re-probe
-            subprocess.call(['udevadm', 'trigger', '--action=change',
-                             '--sysname-match=' + os.path.basename(self.cd_device)])
+            # the scsi_debug CD drive content is the same as for the HD drive,
+            # but udev does not know about this; so give it a nudge to re-probe
+            subprocess.call(
+                ['udevadm', 'trigger', '--action=change',
+                 '--sysname-match=' + os.path.basename(self.cd_device)])
```

Comment:
This and the above function calls with arguments on the next lines are terrible.